### DISCUSSION TITLE
Upgrading to zora 5

### DISCUSSION THREAD

**Discussion by mindplay-dk** [lorenzofox3/zora#99] [General]
Is there an upgrade guide for version 5.0 somewhere?

I'm curious what to expect in terms of breaking changes.

It would be helpful to mention this in the README - or consider adding an `UPGRADING.md` to the project root.

**Comment by lorenzofox3** [comment]
Good point. 

## zora

So far, I would say:
* all the top level assertion do not exist anymore ``import {ok ,etc} from 'zora'``, you need to wrap the assertion in a ``test`` function
* the tap indent reporter does not exist (yet ?)

## pta

Pta (the node  runner) is able to run any zora program and does not require a default export anymore:

```Javascript
// before

export default (t) => {
    t.ok(true
}

// after
import {test} from 'zora';

test('some spec', ({ok}) => {
     ok(true)
})
```
